package.json

```diff
@@ -31,6 +31,9 @@
     "postversion": "git push && git push --tags"
   },
   "types": "dist/index.d.ts",
+  "dependencies": {
+    "eslint-plugin-react": "^7.23.2"
```

Comment:
this should probably be a peerDependency as well.

Reply:
Good catch, thanks